doc/specs/vulkan/chapters/VK_EXT_debug_report.txt

```diff
@@ -223,14 +231,11 @@ include::../api/protos/vkDebugReportMessageEXT.txt[]
   * pname:objectType is a elink:VkDebugReportObjectTypeEXT specifying the
     type of object being used or created at the time the event was
     triggered.
-  * pname:object is object where the issue was detected.
+  * pname:object object where the issue was detected.
     pname:object may be ename:VK_NULL_HANDLE if there is no object
     associated with the event.
-  * pname:location is a component (layer, driver, loader) defined value that
-    indicates the _location_ of the trigger.
-    This is an optional value.
-  * pname:messageCode is a layer-defined value indicating what test
-    triggered this callback.
+  * pname:location is an application defined value.
+  * pname:messageCode is an application defined value.
   * pname:pLayerPrefix is the abbreviation of the component making the
     callback.
```

Comment:
Same problem. Should be something like "making this event/message" instead "making the callback".